independent-projects/arc/pom.xml

```diff
@@ -55,9 +55,9 @@
         <!-- test versions -->
         <version.assertj>3.24.2</version.assertj>
         <version.junit5>5.10.0</version.junit5>
-        <version.kotlin>1.9.10</version.kotlin>
-        <version.kotlin-coroutines>1.7.1</version.kotlin-coroutines>
-        <version.mockito>5.4.0</version.mockito>
+        <version.kotlin>1.9.20</version.kotlin>
+        <version.kotlin-coroutines>1.7.3</version.kotlin-coroutines>
+        <version.mockito>5.7.0</version.mockito>
```

Comment:
Is there a reason why you also upgraded Mockito in this PR?

If so, it would be better to mention it in the PR title too.

Reply:
The newest version also works with Kotlin 1.9.20. So it seemed reasonable to upgrade it too. I will change the title. Thanks for the hint!